guides/common/modules/proc_preparing-for-disaster-recovery-with-two-active-project-servers.adoc

```diff
@@ -0,0 +1,50 @@
+[id="preparing-for-disaster-recovery-with-two-active-project-servers"]
+= Preparing for disaster recovery with two active {ProjectServer}s
+
+Create a second {ProjectServer} by restoring a backup of your first {ProjectServer}.
+Configure both servers to operate independently in their respective data centers, but ensure that their content, such as repositories or host configuration, does not drift apart over time.
+
+.Procedure
+. Back up your {ProjectServer}.
+For more information, see xref:backing-up-{project-context}-server-and-{smart-proxy-context}_admin[].
+. Restore the backup on a system that will serve as your other {ProjectServer}.
+For more information, see xref:restoring-{project-context}-server-or-{smart-proxy-context}-from-a-backup_admin[].
++
+[NOTE]
+====
+Each server must have a distinct hostname and IP address.
+This enables you to re-register hosts if one of the servers fails.
+====
+. Ensure that content on your servers remains consistent:
+* If you want both servers to manage content synchronization and content view creation, follow these guidelines to prevent content drift:
+** Regularly synchronize repositories on both servers.
+For more information, see {ContentManagementDocURL}[Synchronizing repositories] in _{ContentManagementDocTitle}_.
+You can use the following Ansible modules to automate repository synchronization: `{ansible-namespace}.repository_sync` and `{ansible-namespace}.sync_plan`.
+** Ensure that content views on both servers match.
```

Comment:
How?

Reply:
1. Use ISS if the disaster recovery site has network access to the primary
2. Use import-export if there is disconnection between the two
3. Use the dedicated content server approach in either case if a single source of truth is desired, with dedicated control by a system admin that owns the SOE.

Reply:
Added to appropriate places in the existing steps.